psiminer-core/src/main/kotlin/Common.kt

```diff
@@ -24,7 +24,8 @@ enum class Dataset(val folderName: String) {
 
 enum class Language(val extensions: List<String>) {
     Java(listOf("java")),
-    Kotlin(listOf("kt", "kts"))
+    Kotlin(listOf("kt", "kts")),
+    PHP(listOf("php"))
```

Comment:
Well, hard to say because I didn't manage to find any examples of such files. Even the oldest PHP files on GitHub have `.php` extension so I'd rather not support any other formats. If you have another opinion, we can always change it quickly.